docs/src/new_mnemonic.md

```diff
@@ -0,0 +1,26 @@
+# new-mnemonic
+
+## Description
+Generates a new BIP-39 mnemonic along with validator keystore and deposit files depending on how many validators you wish to create.
+
+## Optional Arguments
+
+- **`--mnemonic_language`**: The language of the BIP-39 mnemonic. Options are: 'chinese_simplified', 'chinese_traditional', 'czech', 'english', 'french', 'italian', 'japanese', 'korean', 'portuguese', 'spanish'.
+
+- **`--chain`**: The chain to use for generating the deposit data. Options are: 'mainnet', 'holesky', etc...
+
+- **`--num_validators`**: Number of validators to create.
+
+- **`--keystore_password`**: The password that is used to encrypt the provided keystore. Note: It's not your mnemonic password.
+
+- **`--execution_address`**: The Ethereum 1 address for validator withdrawals.
+
+- **`--pbkdf2`**: Will use pbkdf2 key derivation instead of scrypt for generated keystore files as defined in [EIP-2335](https://eips.ethereum.org/EIPS/eip-2335#decryption-key). This can be a good alternative if you intend to work with a large number of keys.
```

Comment:
Include a small sentence about pbkdf2 behind less secure than scrypt and this option being more like an expert/advanced use of the tool. Maybe something along the lines of *avoid this unless you know what you are doing*.

Reply:
Added a little message there